src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -175,35 +190,281 @@ type SingleBlastJobResultProps = {
   jobId: string;
   species: Species;
   diagramWidth: number;
+  blastDatabase: DatabaseType;
 };
 
+const hitsTableColumns: DataTableColumns = [
+  {
+    width: '0px',
```

Comment:
Wait, are we rendering this at all, but with a width of zero pixels? Why render it then?

Reply:
It was just a leftover of previous changes. I've removed it